css/mobileStyle.css

```diff
@@ -229,7 +229,6 @@ body {
   margin: auto;
```

Comment:
Great design @danielochuba 🤝

- [ ] As stated in the project requirements. kindly stick to the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as much as possible. Kindly implement these changes across desktop and mobile screens.
- [ ] Kindly make sure your design contains two navbars as seen in the given design.
- [ ] Kindly make sure the general background colors, fonts, and font sizes are replicas of the given design.
- [ ] Kindly make sure your design is responsive on both the desktop and the mobile screens. Some parts of the design shrink and disappear on the mobile view.
- Your Main Design
![desktop-view](https://user-images.githubusercontent.com/63917725/219564297-556b8318-912e-4826-8a87-d0004fb09618.PNG)
- Given Design
![Given-design](https://user-images.githubusercontent.com/63917725/219564346-ed5782c7-ea1d-4e33-885d-e2d28155b9c2.PNG)
- Given program
![given-program](https://user-images.githubusercontent.com/63917725/219564387-02adc4d1-fb68-4ffe-8e14-d1d6839c870b.PNG)
- Your Program
![your-program](https://user-images.githubusercontent.com/63917725/219564420-0a82448e-e3ba-45dc-9368-a5a08d4c92c7.PNG)